fe/src/main/java/org/apache/doris/load/routineload/RoutineLoad.java

```diff
@@ -27,15 +28,18 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Map;
+import java.util.Queue;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 import java.util.stream.Collectors;
 
 public class RoutineLoad {
```

Comment:
I think `RoutineLoadManager` is better.

Reply:
Why `Load` not use `LoadManger`?

Reply:
I think RoutineLoadManger is better too.